src/design-guidelines/action-buttons/action-buttons.md

```diff
@@ -0,0 +1,126 @@
+# Action buttons
+
+Action buttons are typically used in dialogs' footers, or in headers. These are buttons indicate to users that they must deliberately choose one action from a set of actions which are presented, for the system to continue or finish a task.
+
+Action buttons must naturally be presented together in one place; within the UI. When several action buttons are presented, their actions are often contradictory; for example **Discard** versus **Save**.
+
+When it comes to details such as placement of action buttons, choice of labels, and adding meaningful graphical details, it's important to follow a few design conventions, explained here:
+
+<br/>
+<br/>
+
+*****
+
+## Placement
+On a user interface which is designed for a left-to-right script like English, action buttons of a footer (for example in a dialog) are typically located at the bottom-right corner.
```

Comment:
a language cannpt be right-to-left. The way it is written can be :) I think that's why they chose to say script:

https://en.wikipedia.org/wiki/Right-to-left_script

But I don't know.

Reply:
Ok 👍

Reply:
In case you liked this suggestion. I think "of" is ok to though.

```suggestion
On a user interface which is designed for a left-to-right script like English, action buttons in a footer (for example in a dialog) are typically located at the bottom-right corner.
```